src/task_4_3.py

```diff
@@ -0,0 +1,28 @@
+"""Дан словарь: {'test': 'test_value', 'europe': 'eur', 'dollar': 'usd', 'ruble': 'rub'}
+Добавить каждому ключу число равное длине этого ключа (пример {‘key’: ‘value’} -> {‘key3’:
+‘value’}). Чтобы получить список ключей - использовать метод .keys()"""
+slovar = {'test': 'test_value',
+          'europe': 'eur',
+          'dollar': 'usd',
+          'ruble': 'rub'}
+kopia = {'test': 'test_value',
+          'europe': 'eur',
+          'dollar': 'usd',
+          'ruble': 'rub'}
+a = list(slovar.keys())
+b = list(slovar.values())
+schetchik = 0
+schetchik_2 = 0
+new_key_2 = ''
+for i in kopia:
```

Comment:
Попробуй сделать без счетчика и без дополнительных списков a и b

```suggestion
for key, value in kopia.items():
```